vscode-extensions/vscode-spring-boot/lib/apiManager.ts

```diff
@@ -0,0 +1,31 @@
+import { commands, Uri } from "vscode";
+import { Emitter, LanguageClient } from "vscode-languageclient/node";
+import { ExtensionAPI } from "./api";
+import { LiveProcessConnectedNotification, LiveProcessDisconnectedNotification } from "./notification";
+
+export class ApiManager {
+    public api: ExtensionAPI;
+    private onDidLiveProcessConnectEmitter: Emitter<string> = new Emitter<string>();
+    private onDidLiveProcessDisconnectEmitter: Emitter<string> = new Emitter<string>();
+
+    public constructor(private client: LanguageClient) {
+        const onDidLiveProcessConnect = this.onDidLiveProcessConnectEmitter.event;
+        const onDidLiveProcessDisconnect = this.onDidLiveProcessDisconnectEmitter.event;
+
+        const COMMAND_LIVEDATA_GET = "sts/livedata/get";
+        const getLiveProcessData = async (query) => {
+            await commands.executeCommand(COMMAND_LIVEDATA_GET, query);
+        }
+
+        // TODO: STS server should send corresponding notification back.
```

Comment:
Corresponding logic has not been implemented in `spring-boot-language-server` yet. As this project is well decoupled, I'm not sure where should be the best place to do that. The remaning work should be:
- define custom notifications in `STS4LanguageClient`
- send them at the place whereever live process connection is established/disconnected.